pages/linux/eject.md

```diff
@@ -0,0 +1,31 @@
+# eject
+
+> Eject cds, floppy disks and tape drives.
+
+- Display the default device:
+
+`eject -d`
+
+- Eject the default device:
+
+`eject`
+
+- Eject a specific device:
+
+`eject {{name_of_device}}`
+
+- Toggle whether a device's tray is open or closed:
+
+`eject {{name_of_device}} -T`
```

Comment:
Shouldn't the command line options come before the positional parameters?

Reply:
It doesn't seem to make any difference, but I'll maybe move them to the start as that's how man pages order them.